pkgs/os-specific/linux/checksec/default.nix

```diff
@@ -33,18 +40,25 @@ stdenv.mkDerivation rec {
   installPhase =
     let
       path = lib.makeBinPath [
-        findutils
+        binutils
+        coreutils
         file
-        binutils-unwrapped
-        sysctl
+        findutils
+        gawk
+        gnugrep
+        gnused
         openssl
+        procps
+        sysctl
+        which
```

Comment:
[elfutils](https://search.nixos.org/packages?channel=23.05&show=elfutils&from=0&size=50&sort=relevance&type=packages&query=eu-readelf) seems to be missing.

There is a hidden command `checksec --debug_report` to check for the needed runtime dependencies. Might be a good idea to add this as a test.